privaterelay/templates/newlanding/a/how-it-works.html

```diff
@@ -3,36 +3,39 @@
 
 <section>
     <div class="container mzp-l-content">
-    <div class="how-it-works-content txt-white flx flx-col al-cntr">
-    <h2 class="how-it-works-header text-center">{% ftlmsg 'how-it-works-headline' %}</h2>
-    <span class="text-center">{% ftlmsg 'how-it-works-subheadline' %}</span>
+    <div class="how-it-works-content">
+    <h2 class="how-it-works-header">{% ftlmsg 'how-it-works-headline' %}</h2>
+    <span class="how-it-works-subheader">Share ⁨Relay⁩ email aliases instead of your real email address to protect your online accounts from hackers.</span>
     <div class="how-it-works-steps mzp-l-columns mzp-t-columns-three">
         <section class="step step-1">
-        <div class="step-illustration step-1"></div>
+        <img src="/static/images/how-it-works/how-it-works-1.svg" class="step-illustration step-1">
         <div class="step-text step-1">
             <h3 class="step-headline ff-Met">{% ftlmsg 'how-it-works-step-1-headline' %}</h3>
             <p class="step-body">
-            <a href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank" rel="noopener" class="text-link">{% ftlmsg 'how-it-works-step-1-link' %}</a>
-            {% ftlmsg 'how-it-works-step-1-copy' %}
+            <a href="https://addons.mozilla.org/firefox/addon/private-relay/" target="_blank" rel="noopener" class="text-link">Download the ⁨Relay⁩ extension for ⁨Firefox⁩.</a>
+             Select the icon that appears on your ⁨Firefox⁩ toolbar to access the sign-in page. Sign in with your ⁨Firefox account⁩ to get started. 
             </p>
         </div>
         </section>
         <section class="step step-2">
-        <div class="step-illustration step-2"></div>
+        <img src="/static/images/how-it-works/how-it-works-2.svg" class="step-illustration step-2">
         <div class="step-text step-2">
             <h3 class="step-headline ff-Met">{% ftlmsg 'how-it-works-step-2-headline' %}</h3>
-            <p class="step-body">{% ftlmsg 'how-it-works-step-2-copy' %}</p>
+            <p class="step-body">
+            As you browse, the ⁨Relay⁩ icon will appear where sites ask for your email address. Select it to generate a new, random address that ends in @relay.firefox.com. ⁨Relay⁩ will forward messages to the primary email address associated with your account.
```

Comment:
Should these be hardcoded, and should this be `@relay.firefox.com` or `@mozmail.com`?

Reply:
Waiting on the strings to be finalized.